src/components/DeleteMessage.js

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components';
+import ScreenReaderOnly from './ScreenReaderOnly';
+
+export default function DeleteMessage({ onConfirmDelete, onCancelDelete }) {
+  return (
+    <>
+      <MessageOverlay>
+        <MessageContent>
+          <h2>Confirm Delete Request</h2>
+          <p>Do you really want to delete this card?</p>
+          <MessageWindowWrapper>
+            <MessageCancelButton type="button" onClick={onCancelDelete}>
+              No <ScreenReaderOnly>cancel delete this card</ScreenReaderOnly>
+            </MessageCancelButton>
+            <MessageDeleteButton type="button" onClick={onConfirmDelete}>
+              Yes <ScreenReaderOnly>confirm delete this card</ScreenReaderOnly>
+            </MessageDeleteButton>
+          </MessageWindowWrapper>
+        </MessageContent>
+      </MessageOverlay>
+    </>
+  );
+}
+
+const MessageOverlay = styled.div`
+  position: fixed;
+  background-color: rgba(49, 49, 49, 0.3);
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
```

Comment:
Just because I like to know: Does this work as well? [Have a look on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/inset)
```suggestion
inset: 0;
```